.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+# Prerequisites
+
+```Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**```
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+## Expected Behavior
+
+```Please describe the behavior you are expecting```
+
+## Current Behavior / Question
+
+```What is the current behavior?```
+
+## Failure Information (for bugs)
+
+**Remove everything after this if it is a question and not a bug report.**
+
+```Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.```
+
+## Steps to Reproduce
+
+```Please provide detailed steps for reproducing the issue.```
+
+1. step 1
```

Comment:
Follow the same casing: use "**S**tep" (with capital S).

Reply:
Yeah sure. I'll fix this.